lib/stagehand/staging/synchronizer.rb

```diff
@@ -1,6 +1,8 @@
 module Stagehand
   module Staging
     module Synchronizer
+      include Stagehand::Database
```

Comment:
Don't include database. We don't want all the methods going into the synchronizer, just call `Database. production_database_versions` to access the desired method.